services/QuillLMS/engines/evidence/app/models/evidence/gen_ai.rb

```diff
@@ -4,6 +4,24 @@ module Evidence
   module GenAI
     HistoryItem = Data.define(:user, :assistant)
 
+    CONCEPT_MAPPING = {
+      Evidence::Activity::BECAUSE_CONJUNCTION => 'qkjnIjFfXdTuKO7FgPzsIg',
+      Evidence::Activity::BUT_CONJUNCTION => 'KwspxuelfGZQCq7yX6ThPQ',
+      Evidence::Activity::SO_CONJUNCTION => 'IBdOFpAWi42LgfXvcz0scQ'
+    }
+
+    RULES_OPTIMAL = {
```

Comment:
So, this is small thing, but when PG requested that these rules not be universal, I think (and I could be wrong) that he wanted a different rule per prompt that each pointed to the same 3 concepts rather than 6 total rules that are re-used. We probably should just clarify this. Personally I think the way you have it is cleaner, but I don't have strong enough feelings about if he has strong feelings about making a new set of rules for each prompt.

Reply:
I confirmed with Peter that we want a different rule per prompt